### DISCUSSION TITLE
Adding a folder/subtag or search is hidden when dynamic panel enabled and mouse outside

### DISCUSSION THREAD

**Discussion by eadoking** [standardnotes/forum#2834] [General]
## The main scope the how to re-produce

When trying to add a folder/task, and have the dynamic panels enabled.  The search panel disappears, and I can continue writing but I don't see what I am writing, and this happens exactly when the Mose move outside the left size or entered the next panel.

## Expected behavior and solution

When adding folder/subtask and having the dynamic panels enabled, the search/ adding panel suppose to be locked (hiding is temporary disabled) until I finish writing or entering the folder/subtask, or search 

## tested on 

MacBook Pro 14, MacOS 12.4, Standard Note v 3.23.77, account type: Pro

**Comment by myreli** [comment]
Can replicate that by interacting with the mouse outside of the panel hides it even if an action is focusing (for example editing tags). 

https://user-images.githubusercontent.com/17554234/220798775-3c57719c-752c-428d-a901-a5a633fa952d.mov
